doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -0,0 +1,1100 @@
+2291    raised       TEMPERATURE_EXCEEDED             Wed Feb 10 18:08:24 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+182     cleared      TEMPERATURE_EXCEEDED             Wed Feb 10 09:52:12 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+134     raised       TEMPERATURE_EXCEEDED             Wed Feb 10 09:32:01 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+        
+sonic# show event details 
+Event History Details - 2292
+-------------------------------------------
+Sequence Number:   2292
+Severity:          informational
+Name:              TAM_SWITCH_ID_CHANGE
+Description:       processSwitchTblEvent: Received new switch-id as 3
+Timestamp:         Wed Feb 10 18:08:27 2021
+Source:            3
+State:             -
+-------------------------------------------
+Event History Details - 2291
+-------------------------------------------
+Sequence Number:   2291
+Severity:          critical
+Name:              TEMPERATURE_EXCEEDED
+Description:       temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+Timestamp:         Wed Feb 10 18:08:24 2021
+Source:            sensor/2
+State:             raised
+
+sonic# show event summary
+
+Event Summary
+-------------------------------------------
+Raised:                 5
+Ack:                    1
+Cleared:                4
+Events:                 20
+-------------------------------------------
+
+sonic# show event severity critical
+Sq No   State        Name                             Timestamp                   Description
+-----   --------     ------------------------------   --------------------------- --------------
+2291    raised       TEMPERATURE_EXCEEDED             Wed Feb 10 17:58:19 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees. 
+
+sonic# show event recent 5min 
+Sq No   State        Name                             Timestamp                   Description
+-----   --------     ------------------------------   --------------------------- --------------
+2292    -            TAM_SWITCH_ID_CHANGE             Wed Feb 10 18:08:27 2021    processSwitchTblEvent: Received new switch-id as 3        
+2291    raised       TEMPERATURE_EXCEEDED             Wed Feb 10 18:08:24 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
+
+sonic# show event sequence-id from 2290 to 2300
+Sq No   State        Name                             Timestamp                   Description
+-----   --------     ------------------------------   --------------------------- --------------
+2292    -            TAM_SWITCH_ID_CHANGE             Wed Feb 10 18:08:27 2021    processSwitchTblEvent: Received new switch-id as 3        
+2291    raised       TEMPERATURE_EXCEEDED             Wed Feb 10 18:08:24 2021    temperatureCrossedThreshold: Current temperature of sensor/2 is 76 degrees
```

Comment:
Suggest to have the State similar in all show commands. Better to have it started with capital letter.

Reply:
We may cross 80-column limit. Will explore.

Reply:
All show commands are updated with state names uniform (raised, cleared, acknowledged). The 'show event status' command displays counters corresponding to states hence they are named as Raised, Cleared and Acknowledged.